UnitTests/UnitTests.XamlIslands.Package/Package.appxmanifest

```diff
@@ -0,0 +1,49 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  IgnorableNamespaces="uap rescap">
+
+  <Identity
+    Name="e9b2d6ff-2c1a-456b-9ebb-813a54796bc9"
+    Publisher="CN=Microsoft"
+    Version="1.0.0.0" />
+
+  <Properties>
+    <DisplayName>UnitTests.XamlIslands.Package</DisplayName>
+    <PublisherDisplayName>Microsoft</PublisherDisplayName>
+    <Logo>Images\StoreLogo.png</Logo>
+  </Properties>
+
+  <Dependencies>
+    <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.0.0" MaxVersionTested="10.0.0.0" />
+    <TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.14393.0" MaxVersionTested="10.0.14393.0" />
```

Comment:
Min and Max versions tested are too low here, islands only works on 19H1+

Reply:
Addressed by https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3206/commits/919df39bd515b7fe7260042fd5407cdd912ce491